From df95fd51a1a5c77ddce1cd35e59c7ae562367c58 Mon Sep 17 00:00:00 2001
From: MSe1969 <mse1969@posteo.de>
Date: Mon, 19 Oct 2020 20:02:00 +0200
Subject: [PATCH] Alter whitelist to v2-signed F-Droid client

In order to fix the vulnerable v1 certificate of the F-Droid client, it has
been build from source and signed with my specific build release key.
Hence, the hard-coded whitelist must be altered accordingly.
---
 app/src/main/java/org/fdroid/fdroid/privileged/ClientWhitelist.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/app/src/main/java/org/fdroid/fdroid/privileged/ClientWhitelist.java b/app/src/main/java/org/fdroid/fdroid/privileged/ClientWhitelist.java
index 993fadb..25395d0 100644
--- a/app/src/main/java/org/fdroid/fdroid/privileged/ClientWhitelist.java
+++ b/app/src/main/java/org/fdroid/fdroid/privileged/ClientWhitelist.java
@@ -35,7 +35,7 @@ public class ClientWhitelist {
 
     public static HashSet<Pair<String, String>> whitelist = new HashSet<>(Arrays.asList(
             // certificate SHA-256 of https//f-droid.org/F-Droid.apk
-            new Pair<>("org.fdroid.fdroid", "43238d512c1e5eb2d6569f4a3afbf5523418b82e0a3ed1552770abb9a9c9ccab")
+            new Pair<>("org.fdroid.fdroid", "6c410a51720a1f543e5b078e1e4ea661ac03421f8adbd182f27ef93eb6198355")
     ));
 
 }
-- 
2.11.0

